runtime/gc_realtime/RealtimeMarkingSchemeRootMarker.hpp

```diff
@@ -44,7 +44,7 @@
  */
 class MM_RealtimeMarkingSchemeRootMarker : public MM_RealtimeRootScanner
 {
-/* Data memebers / types */
+/* Data members / types */
```

Comment:
this change used to be in runtime/gc_realtime/RealtimeMarkingScheme.cpp before the conflict